schema/Mutation.graphql

```diff
@@ -84,4 +84,10 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create new lives
+    """
+    createLife(life: createLifeInput): Life!
```

Comment:
`createLifeInput` should be mandatory. This helps you to know that you need to have `life` so you can call this mutation